test/integration/consul-container/test/README.md

```diff
@@ -0,0 +1,67 @@
+# Upgrade Integration Tests
+
+The upgrade integration test is responsible for ensuring that users are able to upgrade between consul versions without experiencing difficulties. The upgrade tests does the following:
+- Ensure that customers are able to upgrade to a new Consul version and new and existing features work as expected. 
+- Test for backwards compatibility. i.e support for upgrading directly from Consul `1.13` to `1.15`, without upgrading to `1.14`
+    - Highlights errors that will occur if attempting to skip a version during upgrades. Example the API changed between version 1.13 and 1.15 so direct upgrades between these version is not supported. 
```

Comment:
I don't quite understand this point, could you please elaborate a bit? Thanks.

Reply:
I am describing here the importance of the upgrade tests. The upgrade tests will highlight errors that may occur as customers attempt to upgrade their current version to the latest version. I was then trying to give an example of the API data structure change that prevents users from upgrading from 1.12.3 -> 1.13.